package.json

```
@@ -26,14 +26,15 @@
     "lint-fix:markdown": "npm run lint:markdown -- --write",
     "package:deb": "electron-packager . time-to-leave --overwrite --asar --platform=linux --arch=x64 --icon=assets/ttl.png --prune=true --out=release-builds --ignore=__tests__ --ignore=tests --ignore=scripts --ignore=Resources --ignore=website --ignore=release-builds --ignore=packages --ignore=coverage_jest --ignore=.nyc_output --ignore=__mocks__ --ignore=docs",
     "package:mac": "electron-packager . 'Time to Leave' --overwrite --platform=darwin --arch=x64 --icon=assets/icon-mac.icns --prune=true --out=release-builds --ignore=__tests__ --ignore=tests --ignore=scripts --ignore=Resources --ignore=website --ignore=release-builds --ignore=packages --ignore=coverage_jest --ignore=.nyc_output --ignore=__mocks__ --ignore=docs",
-    "package:win": "electron-packager . time-to-leave --overwrite --platform=win32 --arch=ia32 --icon=assets/ttl.ico --prune=true --out=release-builds --version-string.CompanyName=CE --version-string.FileDescription=CE --version-string.ProductName=\"Time to Leave\" --ignore=__tests__ --ignore=tests --ignore=scripts --ignore=Resources --ignore=website --ignore=release-builds --ignore=packages --ignore=coverage_jest --ignore=.nyc_output --ignore=__mocks__ --ignore=docs",
+    "package:win": "electron-packager . \"Time to Leave\" --overwrite --platform=win32 --arch=ia32 --icon=assets/ttl.ico --prune=true --out=release-builds --version-string.CompanyName=CE --version-string.FileDescription=CE --version-string.ProductName=\"Time to Leave\" --ignore=__tests__ --ignore=tests --ignore=scripts --ignore=Resources --ignore=website --ignore=release-builds --ignore=packages --ignore=coverage_jest --ignore=.nyc_output --ignore=__mocks__ --ignore=docs",
     "preinstall": "npx npm-force-resolutions",
     "postinstall": "patch-package",
     "pretest": "npm run clean",
     "start": "electron .",
     "test": "npm-run-all test:*",
     "test:jest": "jest --runInBand --detectOpenHandles --verbose --colors",
-    "test:mocha": "nyc --reporter=lcov --reporter=json mocha tests/*"
+    "test:mocha": "nyc --reporter=lcov --reporter=json mocha tests/*",
+    "setup" : "npx electron-installer-windows --src ./release-builds/\"Time to Leave-win32-ia32\"/ --dest ./release-builds/\"Time to Leave-win32-ia32\"/installers/ --exe \"Time to Leave.exe\""
```

```suggestion
    "setup:win" : "npx electron-installer-windows --src ./release-builds/\"Time to Leave-win32-ia32\"/ --dest 'packages\' --verbose --exe \"Time to Leave.exe\""
```
I suggest renaming this to `setup:win` and reusing it on `.github\workflows\Release.yml`. I brought the `dest` configuration from there.

Also, any reason for naming the exe explicitly? Would it be "time-to-leave" if not?

---

If we use "Time to Leave" for application name, then it will create Time to Leave.exe. Default is, according to .nuspec, time-to-leave.exe. Thus, if I do not specify, it said it failed on finding corresponding .exe file. 
Also, is there a way to test release? I want to make sure my release process  works correctly.

---

I'm not sure there's a way to test the release without actually releasing it here. I haven't played with the actions yet.
Maybe you can run the actions in your fork?

---

Thank you for the suggestion. I was able to run the action in my fork and it passed.

---

Great!